conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
+# All Rights Reserved
+#
+# conda is distributed under the terms of the BSD 3-clause license.
+# Consult LICENSE.txt or http://opensource.org/licenses/BSD-3-Clause.
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from .core import repodata as _impl
+from .connection import CondaSession
+from os.path import join
+
+
+class RepoData(object):
```

Comment:
Please review for yourself (and as a general code review also) the `PackageCache` class.  Just like this class, it is managing a collection of collections.  That is to say, there are multiple package caches, each package cache being a collection of package cache entries (should be renamed to package cache records).  The parallel is that the RepoData class manages multiples repodatas (for lack of a better word), and each repodata is a collection of repodata records.